ts/src/base/Exchange.ts

```diff
@@ -1760,6 +1760,18 @@ export default class Exchange {
         return parseInt (convertedNumber);
     }
 
+    parseToNumeric (number) {
```

Comment:
@Dan-krm I think this method will parse the number to the desired type in python and PHP